libs/blocks/preflight/panels/assets.js

```diff
@@ -17,7 +21,19 @@ const viewportTooSmall = signal(isViewportTooSmall());
  * Runs asset checks and updates signals with the results.
  */
 async function getResults() {
-  const checks = runChecks(window.location.pathname, document);
+  let checks;
```

Comment:
You can pretty much keep the whole code as it is and just call
`const { isViewportTooSmall, runChecks } = await preflightApi.getResults();`